docs/teams.md

```diff
@@ -16,8 +16,8 @@ export class TeamAwesome extends ApplicationTeam {
             users: [
                 new ArnPrincipal(`arn:aws:iam::${YOUR_IAM_ACCOUNT}:user/user1`),  
                 new ArnPrincipal(`arn:aws:iam::${YOUR_IAM_ACCOUNT}:user/user2`)
-            ]
-
+            ],
+            policyDir: './examples/teams/team-awesome/'
```

Comment:
Nice, very simple and precise. We can also think about a URL here as well  if it is not too much work (e.g. github repo URL or something like that). Can be deferred.